### PR TITLE
Add a plot tool to draw animation and modify the code style

### PR DESCRIPTION
**Pull request by xueminchi**
I add a utlis_tutorial file to manage the 'plot functions' which will be used in existed tutorials and future tutorials. The features around the visualization on tutorials could be managed and modified in this file. 
Later, I want to move and do some modification to the 'plot trajectory', 'plot multi trajectory', 'plot energies' functions. I will add 'arrows', and animation functions like shown in this link:https://www.youtube.com/watch?v=_7B181kS4Xg&t=2339s at time 18 mins 50 seconds. 

I also made the code style to PEP8 sytle because I used Pycharm IDE, sometimes I have the habbit to do this automatically. I don't know if it will cause troubles for you. 

When I work at home, I only have a laptop, the resolution is too high, so I have to tune the 'figure_dpi' to 200, the original is 100. I add this choice to the utils_tutorial. 

I review the two questions you aksed Nathan previously, I think you push the theory part from stationary environments to dynamic environments, there are many concepts in optimzation fabrics can't effectly explain the some stuff in dynamic environments. Like the 'path consistency', 'path' in my understanding is without time parameterization, otherwise it would be called 'trajectory'. To solidate the theory in dynamic environments, I will be interested to read some papers around other reactive methods to see how they handle some difficult setups.

My plan to this tutorial is to reproduce the original results as much as possible based on your current version. I will also pay attention to the videos and resources released by Nathan's group. On the other hand, I will be careful because when I read a paper from a standford phd student, he mentioned some conclusions in RMPflow are wrong and validate it. 

The second step is to add tutorial related to dynamics fabrics and some tutorials like the collision avoidance you provided in jupyter notebook. Instead of coding directly, using fabrics package to develop more efficiently.

### CODE REVIEW

**Review by maxspahn — CHANGES_REQUESTED** [review]
Hi @xueminchi ,

Thanks a lot for this initial PR. I am really glad you want to help with this repository.
I think the goal of outsourcing the animate function is to use that one in all the tutorial files. I agree with that idea. This PR should include that. If you need help with the other tutorial files, let me know. I made some comments to the code, see the review. For future PR's: It is better to create a branch for each specific PR, so you don't pollute your own develop branch.

Best,
@maxspahn

File: tutorial/utlis_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+plt.rcParams['figure.dpi'] = 200                    # figure dpi
+plt.rcParams["font.family"] = "Times New Roman"     # font style
```
[the remark this answers is not in the document]
Comment (xueminchi) [reply tutorial/utlis_tutorial.py:5]: Thanks for pointing out this, and I agree with that. The 'Times News Roman' is indeed not absolutely need.

**Comment by xueminchi** [comment]
Really thanks for reviewing the codes! It helps a lot. I will create a branch for each specific PR next time. Previously, I was not sure about this because it's the first PR, now I know and thank you so much.

**Comment by maxspahn** [comment]
This PR could potentially be merged, but I think you wanted to do more changes before we merge it, right? Let me know what your next steps are.

**Comment by xueminchi** [comment]
> This PR could potentially be merged, but I think you wanted to do more changes before we merge it, right? Let me know what your next steps are.

Yes. Around this tutorial, my next step it to move visualization functions like trajectories, energies to 'utlis_tutorial'. Then modify the current tutorials in section 10 to parallel version (on CPU not GPU) and add stuffs like arrows as shown last time. I'm not sure if the parallel is necessary, it just saves time to compute with a group of initial states.

**Comment by xueminchi** [comment]
> This PR could potentially be merged, but I think you wanted to do more changes before we merge it, right? Let me know what your next steps are.

I guess I could close this pull request and create a feature branch to contribute. When it's done, I will initiate another pull request.

**Pull request closed without merging**